.github/workflows/regression-tests.yml

```diff
@@ -256,7 +256,8 @@ jobs:
     - name: Install cocotb (Windows, mingw)
       if: startsWith(matrix.os, 'windows') && matrix.toolchain == 'mingw'
       run: |
-        python -m pip install --global-option build_ext --global-option --compiler=mingw32 -v -e .[bus]
+        python ./setup.py build_ext --compiler=mingw32 develop
```

Comment:
An alternative that may work (I did not check) is:

```bash
(echo [build] & echo compiler = mingw32) > %CONDA_PREFIX%\Lib\distutils\distutils.cfg
```

https://python-at-risoe.pages.windenergy.dtu.dk/compiling-on-windows/configuration.html

Reply:
https://wiki.python.org/moin/WindowsCompilers

Reply:
That would work, I saw that as well. But it modifies the python installation, which I'm not crazy about.

Reply:
OK